django/contrib/admin/views/autocomplete.py

```diff
@@ -24,11 +25,18 @@ def get(self, request, *args, **kwargs):
             return JsonResponse({'error': '403 Forbidden'}, status=403)
 
         self.term = request.GET.get('term', '')
+        self.to_field = request.GET.get('to_field', None)
+        if self.to_field:
+            if not self.model_admin.to_field_allowed(request, self.to_field):
+                raise DisallowedModelAdminToField("The field %s cannot be referenced." % self.to_field)
+        else:
+            self.to_field = 'pk'
```

Comment:
You could already set `to_field` to `pk` the `get` method in line 28, could you not?